### PR TITLE
Add helper script to create LVM thin pool for Docker/serviced use

### PR DESCRIPTION
**Pull request by iancmcc**

### CODE REVIEW

File: create-device-thinpool.sh [hunk 1]
```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+set -e
+
+usage() {
+    >&2 echo "Usage: create-devicemapper-thinpool.sh (docker|serviced) /dev/XXX [/dev/YYY...]"
```
Comment (tgjonestx) [inline create-device-thinpool.sh]: nitpick - the name of the file and filename in the Usage statement are different

File: create-device-thinpool.sh [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                echo "Please add the following to /etc/default/docker, stop Docker, delete /var/lib/docker, and start it again:"
+                echo
+                echo DOCKER_OPTS=\"\${DOCKER_OPTS} --storage-opt dm.thinpooldev=${TPOOL_NAME}\"
+                echo 
+                ;;
+            serviced)
+                echo "Please add the following to /etc/default/serviced:"
+                echo
+                echo SERVICED_DM_THINPOOLDEV=\"${TPOOL_NAME}\"
+                echo
```
Comment (tgjonestx) [inline create-device-thinpool.sh]: Should also include echo statements that tell the user to add `SERVICED_FS_TYPE=devicemapper`

Comment (tgjonestx) [inline create-device-thinpool.sh]: When I ran the script in an HA cluster (not sure if that's an issue or not), it told to set the serviced option as:
```
SERVICED_DM_THINPOOLDEV="/dev/mapper/serviced-serviced--pool-tpool"
```
But serviced would not start with that value. The error in the log was:
```
F1104 14:00:05.206571 07936 daemon.go:314] Could not initialize storage driver type=devicemapper root=/opt/serviced/var/volumes args=[dm.thinpoold
ev=/dev/mapper/serviced-serviced--pool-tpool] options=map[]: Non existing device serviced-serviced--pool-tpool
```
Instead I had to use this value which did work
```
SERVICED_DM_THINPOOLDEV="/dev/mapper/serviced-serviced--pool"
```

**Comment by tgjonestx** [comment]
PS. I didn't have a chance to try using the script to setup docker.

**Comment by tgjonestx** [comment]
Given the recent additions to the serviced-storage tool, should we close this PR?

**Comment by wwwtyro** [comment]
Reopen if we still need this. Seems like we don't.

**Pull request closed without merging**